cluster/src/main/java/com/hartwig/pipeline/execution/vm/ComputeEngine.java

```diff
@@ -42,6 +46,7 @@
 
 public class ComputeEngine implements CloudExecutor<VirtualMachineJobDefinition> {
     private final static String APPLICATION_NAME = "vm-hosted-workload";
+    private final static int NUMBER_OF_PERSISTENT_SSDS = 4;
```

Comment:
Why 4? I guess 3 would already give us the 1TB of space? 

Also think the name is wrong as these are not persistent ssds right? I think a good name would be ONE_TB_LOCAL_SSD or something like that.

Reply:
Yeah bad naming. Also the constant in `BashStartupScript` should be shared with this one, I just realised.

I didn't comment this because we generally don't, but there were a few counterintuitive things that came up when I was getting the local SSDs working. One of them was this: if you try to assign 3 it will error out at provisioning time. I can find the reference if you want it; from memory it was to do with the number of cores provisioned. So I had to choose between 2 and 4. If cost is a real issue and we can confirm there's no operational impact, we could drop down to 2 x 375GB.